docs/patterns/navigation.md

```diff
@@ -18,8 +18,12 @@ displayed horizontally on larger screens.
   </p>
 </div>
 
-The background color of a navigation pattern can be set via the
-`$color-navigation-background` variable.
+The navigation pattern is on of the firsts to implement the new theming architecture in vanilla. By default, the nav is light. t`o switch to a dark nav, you can either:
```

Comment:
"The navigation pattern is one of the first patterns to implement the new theming architecture in Vanilla. The default is light. But, to switch to a dark navigation, you can either:"